content/en/apps/examples/pih.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "YendaNafe CHT app by PIH in Malawi"
+linkTitle: "PIH Malawi"
+weight: 
+description: >
+  The YendaNafe app co-designed by PIH Malawi and Medic
+keywords:  
+---
+
+Since 2017, PIH Malawi and Medic have collaboratively developed the CHT-based YendaNafe app, a digital health application for community-based health service provision and coordination. Now, in the spirit of radical collaboration and openness, PIH and Medic have coordinated the release of the full application source code of YendaNafe as first of its kind “Integrated CHT App”, free for all to access and replicate.
```

Comment:
Would be good to spell out PIH as Partners In Health at least once on the page, either in the description or first paragraph, and include an appropriate link, like pih.org or pih.org/country/malawi

Reply:
Updated